### PR TITLE
use nethserver-nextcloud wrapper to configure nextcloud

### PR DESCRIPTION
**Pull request by markVnl**
NethServer/dev#6426

cc/ @mrmarkuz @Amygos

### CODE REVIEW

File: root/etc/e-smith/events/actions/nethserver-collabora-conf [hunk 1]
```diff
@@ -44,11 +44,11 @@ if [[ -n `config getprop loolwsd VirtualHost` ]]; then
   #Configure Nextcloud
   if [[ -x "/usr/share/nextcloud/occ" ]]; then
 
-    sudo -u apache scl enable rh-php72 "/usr/share/nextcloud/occ app:install richdocuments"
+    sudo -u apache scl enable rh-php73 "/usr/share/nextcloud/occ app:install richdocuments"
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethserver-collabora-conf]:
```suggestion
    /usr/local/sbin/occl richdocuments
```

File: root/etc/e-smith/events/actions/nethserver-collabora-conf [hunk 2]
```diff
@@ -44,11 +44,11 @@ if [[ -n `config getprop loolwsd VirtualHost` ]]; then
...
 
-    sudo -u apache scl enable rh-php72 "/usr/share/nextcloud/occ config:app:set richdocuments wopi_url --value=https:\/\/`config getprop loolwsd VirtualHost`"
+    sudo -u apache scl enable rh-php73 "/usr/share/nextcloud/occ config:app:set richdocuments wopi_url --value=https:\/\/`config getprop loolwsd VirtualHost`"
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethserver-collabora-conf]: I remove also the slash escape, but 'm not sure this work, please just try it on machine where collabora is instelled:
```suggestion
    /usr/local/sbin/occ config:app:set richdocuments wopi_url --value=https://`config getprop loolwsd VirtualHost`
```

File: root/etc/e-smith/events/actions/nethserver-collabora-conf [hunk 3]
```diff
@@ -44,11 +44,11 @@ if [[ -n `config getprop loolwsd VirtualHost` ]]; then
...
 
-    sudo -u apache scl enable rh-php72 "/usr/share/nextcloud/occ app:enable richdocuments"
+    sudo -u apache scl enable rh-php73 "/usr/share/nextcloud/occ app:enable richdocuments"
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethserver-collabora-conf]:
```suggestion
    /usr/local/sbin/occ app:enable richdocuments
```

**Review by gsanchietti — CHANGES_REQUESTED** [review]
As future-proof change, use the new occ wrapper which should take care of php version

File: root/etc/e-smith/events/actions/nethserver-collabora-conf (hunk 2 above)
Comment (markVnl) [reply root/etc/e-smith/events/actions/nethserver-collabora-conf]: will test this evening.

**Comment by markVnl** [comment]
First test: installation worked

File: root/etc/e-smith/events/actions/nethserver-collabora-conf (hunk 2 above)
Comment (markVnl) [reply root/etc/e-smith/events/actions/nethserver-collabora-conf]: Install was oke

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**